cluster/gce/addons/konnectivity-agent/konnectivity-agent-ds.yaml

```diff
@@ -48,8 +48,9 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
           resources:
-            limits:
+            requests:
               cpu: 50m
+            limits:
```

Comment:
Was it intentional to change the `cpu` to `requests` but keeping the `memory` as `limits`?

Reply:
Yes, intentional. I wanted to avoid artificial restrictions from cpu limit, as well as incrementally stay similar to GKE configuration.

Reply:
Are we experiencing bursty CPU behavior on the agent?

Reply:
> Are we experiencing bursty CPU behavior on the agent?

No, this is just a best practice / cleanup (this PR shrank after a rebase but this seems worth keeping).

Reply:
+1 for this change.

Some background on the best practices, just in case: CPU limits, due to some bugs in the kernel in several versions, create throttling when sometimes is not needed. Given the way CFS works, also, when no cpu limits is used and there is contention, cpu will be allocated proportionally to the requests. No cpu limits is quite fine :)

Reply:
And mem limits work fine and you want them. So, changing only cpu limits to requests LGTM